test/model/message_list_test.dart

```diff
@@ -311,6 +311,179 @@ void main() {
     check(model).fetched.isFalse();
   });
 
+  group('UserTopicEvent', () {
+    // The ChannelStore.willChangeIfTopicVisible/InStream methods have their own
+    // thorough unit tests.  So these tests focus on the rest of the logic.
+
+    final stream = eg.stream();
+    const String topic = 'foo';
```

Comment:
nit: Is it intentional that `topic` is only used in `setVisibility`? It might be helpful to reuse it to indicate elsewhere in the tests that this particular topic is affected by the helper.

Reply:
Yeah, thanks — that would be clearer.